include/crypto_structs.h

```diff
@@ -168,25 +174,27 @@ typedef struct
 } SDLS_KEYV_RPLY_BLK_t;
 #define SDLS_KEYV_RPLY_BLK_SIZE (sizeof(SDLS_KEYV_RPLY_BLK_t))
 
+
 typedef struct
 {
-    SDLS_KEYV_RPLY_BLK_t blk[29]; // Key Verification Reply Block
+    SDLS_KEYV_RPLY_BLK_t blk[SDLS_KEY_VERIFY_BLK_SIZE]; // Key Verification Reply Block
 } SDLS_KEYV_RPLY_t;
 #define SDLS_KEYV_RPLY_SIZE (sizeof(SDLS_KEYV_RPLY_t))
 
 typedef struct
 {
     uint16_t kid : 16; // Key ID
-    uint8_t challenged[10];
+    uint8_t challenged[SDLS_CHALLENGED_SIZE];
 } SDLS_KEYDB_CMD_t;
 #define SDLS_KEYDB_CMD_SIZE (sizeof(SDLS_KEYDB_CMD_t))
 
+
 typedef struct
 {
     uint16_t kid : 16;      // Key ID
     uint8_t iv[IV_SIZE];    // Initialization Vector
-    uint8_t challenged[10]; // Encrypted Challenge
-    uint8_t cmac[4];        // Challenge Message Authentication Code
+    uint8_t challenged[SDLS_CHALLENGED_SIZE]; // Encrypted Challenge
```

Comment:
Circa line 196... should this be 'CHALLENGE' instead of 'CHALLENGED'?

Reply:
It's part of the reply struct, so I think 'challenged' is correct on line 196. Maybe line 187 should be changed to 'challenge'